templates/add-ons/documentation_starter/{{cookiecutter.__folder_name}}/docs/glossary.md

```diff
@@ -0,0 +1,74 @@
+---
+myst:
+  html_meta:
+    "description": "Terms and definitions used throughout the Plone Sphinx Theme documentation."
+    "property=og:description": "Terms and definitions used throughout the Plone Sphinx Theme documentation."
+    "property=og:title": "Glossary"
+    "keywords": "Plone, documentation, glossary, term, definition"
+---
+
+This glossary provides example terms and definitions relevant to **{{ cookiecutter.__folder_name }}**. {{ cookiecutter.description }}
+
+```{note}
+This is an example glossary demonstrating MyST Markdown’s `{glossary}` directive. You can adapt it for your project’s appendix by editing or replacing these entries with your own terms and definitions.
+```
+
+(glossary-label)=
+
+# Glossary
+
+```{glossary}
+:sorted: true
```

Comment:
It's kind of a random assortment of terms for the example glossary. Should we cut it down to just one or two that are more likely to be relevant (like "Plone" and "add-on")?

Reply:
The note at the top says these are examples. I added and removed a few terms in d5c68ddf1a8ccaf4d777babd8d6c30e109607a51.